src/behat/Api/Context/ApiContext.php

```diff
@@ -504,4 +504,97 @@ function() use ($host, $service, &$hostId, &$serviceId) {
 
         return [$hostId, $serviceId];
     }
+
+    /**
+     * Send a downtime to a service
+     *
+     * @When I send a downtime on my service :service from host :host
+     */
+    public function iSendDowntimeOnServiceFromHost(string $service, string $host)
```

Comment:
these methods shouldn't be added to the apiContext, but in the centreon downtime feature context